snapcraft/internal/lxd.py

```diff
@@ -17,9 +17,14 @@
 
 import logging
 import os
+import sys
 from contextlib import contextmanager
-from subprocess import check_call, CalledProcessError
+from subprocess import CalledProcessError
 from time import sleep
+import pylxd
+from six.moves.urllib import parse
```

Comment:
Seeing six in python3-only code made me double-take. Would `urllib.parse` be better here?